examples/kotlin-diktat/warn-dir/chapter4/save.toml

```diff
@@ -0,0 +1,12 @@
+[general]
```

Comment:
See https://github.com/diktat-static-analysis/save/pull/323/#discussion_r757358217 about this test

Reply:
Is this a link to the same pr?

Reply:
I thought that we have one test for each mod. No line specified, separator and multiline warning

Reply:
> 

Lol, it was supposed to be a link to a comment :) this one: https://github.com/diktat-static-analysis/save/pull/323#discussion_r757358217

Reply:
> I thought that we have one test for each mod. No line specified, separator and multiline warning

Yeah, but they don't differ that much from each other. Now we create a lot of really similar tests in examples. Here you only change extraction logic, and it can be safely tested separately.